DOCS-2993: add templates for API operations (#28)
MultiSafepay/openapi: docs/variable-schema-template.md

> # This is a template for defining a component with schema type: variable, that is OpenAPI compliant.

Reviewer: ```suggestion
# This is an OpenAPI template for a schema type: variable component.
```
